### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by ben-mulhern**
Couple of typos spotted

### CODE REVIEW

File: content/prompts/no-excessive-dom.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ Always try to avoid superflous markup, in particular you should avoid adding DOM
 
 ## An example
 
-Consider a form label, which will default to being an `inline` element. This means that when rendered, the label will appear to the left of any proceeding input element;
+Consider a form label, which will default to being an `inline` element. This means that when rendered, the label will appear to the left of any preceding input element;
```
Comment (mikeyhogarth) [inline content/prompts/no-excessive-dom.mdx:12]: "Preceding" I think implies "before" but we're talking here about input labels that appear AFTER the label
```
label: input
```
Although I do concede that "proceeding" doesn't sound right either - maybe just "adjacent"?

**Review by mikeyhogarth — CHANGES_REQUESTED** [review]
See comments

File: content/prompts/no-excessive-dom.mdx (hunk 1 above)
Comment (ben-mulhern) [reply content/prompts/no-excessive-dom.mdx:12]: Ah good shout, I totally missed that, thought it was just a typo. How about "following" or "subsequent" ?

Comment (mikeyhogarth) [reply content/prompts/no-excessive-dom.mdx:12]: Either of those is absolutey fine dude, cheers

**Pull request merged**